api/src/main/java/io/opencensus/metrics/Summary.java

```diff
@@ -185,8 +177,11 @@ public static ValueAtPercentile create(double percentile, double value) {
     }
   }
 
-  private static Snapshot copySnapshot(Snapshot snapshot) {
-    return Snapshot.create(
-        snapshot.getCount(), snapshot.getSum(), snapshot.getValueAtPercentiles());
+  private static void checkCountAndSum(Long count, Double sum) {
```

Comment:
`count` and `sum` should be `@Nullable`.

Reply:
done.